src/anybody.js

```diff
@@ -594,7 +630,44 @@ export class Anybody extends EventEmitter {
       }
     }
   }
+
+  calculateStats = () => {
+    // n.b. this needs to match the contract in check_boost.cjs
+    const BODY_BOOST = [
+      0, // 0th body, just for easier indexing
+      0, // 1st body
+      0, // 2nd body
+      1, // 3rd body
+      2, // 4th body
+      4, // 5th body
+      8, // 6th body
+      16, // 7th body
+      32, // 8th body
+      64, //9th body
+      128 // 10th body
+    ]
+
+    const SPEED_BOOST = [
+      1, // <10s left
+      1, // <20s left
+      1, // <30s left
+      1, // <40s left
+      2, // <50s left
+      4 // < 60s left
+    ]
```

Comment:
Just updated contract with these values (#110)

```suggestion
    const SPEED_BOOST = [
      6, // <10s left
      5, // <20s left
      4, // <30s left
      3, // <40s left
      2, // <50s left
      1 // < 60s left
    ]
```